webpack.config.js

```diff
@@ -78,50 +78,50 @@ const config = {
             'react': 'preact/compat',
             'react-dom/test-utils': 'preact/test-utils',
             'react-dom': 'preact/compat',
-            '~': path.resolve(__dirname, 'src/app'),
+            '~': path.resolve(__dirname, 'src/app')
         },
         modules: [
             path.resolve(__dirname, 'src/app'),
-            'node_modules',
+            'node_modules'
         ],
         extensions: ['.js', '.jsx']
     },
     watchOptions: {
         aggregateTimeout: 500,
-        poll: 1000,
+        poll: 1000
     },
     devServer: {
-      client: {
-          logging: 'warn',
-      },
-      devMiddleware: {
-          index: false,
-          publicPath,
-          stats: 'errors-only',
-      },
-      headers: {
-          'Access-Control-Allow-Origin': '*',
-          'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, PATCH, OPTIONS',
-          'Access-Control-Allow-Headers': 'X-Requested-With, content-type, Authorization',
-      },
-      historyApiFallback: { index: `${publicPath}/index.html` },
-      hot: true,
-      liveReload: true,
-      open: true,
-      port: devServerPort,
-      // fonts are always loaded with CORS but the CMS doesn't set CORS headers for them
-      // to get around this, we proxy them instead
-      proxy: {
-          context: (path) => path.startsWith('/cms/assets/fonts'),
-          target: API_ORIGIN,
-          changeOrigin: true
-      },
-      static: {
-          directory: path.resolve(__dirname, './src'),
-          publicPath: publicPath
-      }
-    },
-}
+        client: {
+            logging: 'warn'
+        },
+        devMiddleware: {
+            index: false,
+            publicPath,
+            stats: 'errors-only'
+        },
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Methods': 'GET, POST, PUT, DELETE, PATCH, OPTIONS',
+            'Access-Control-Allow-Headers': 'X-Requested-With, content-type, Authorization',
+        },
+        historyApiFallback: { index: `${publicPath}/index.html` },
+        hot: true,
+        liveReload: true,
+        open: true,
+        port: devServerPort,
+        // fonts are always loaded with CORS but the CMS doesn't set CORS headers for them
+        // to get around this, we proxy them instead
+        proxy: {
+            context: (p) => p.startsWith('/cms/assets/fonts'),
+            target: API_ORIGIN,
+            changeOrigin: true
+        },
+        static: {
+            directory: path.resolve(__dirname, './src'),
+            publicPath
+        }
+    }
+};
 
```

Comment:
@RoyEJohnson do these lint changes seem correct to you?  it seems like it some of the removals might be undesired, such as the trailing commas.

Reply:
I don't generally use trailing commas in osweb. Run `yarn eslint` and `yarn stylelint` to ensure everything is still happy. The build system doesn't always error out on style errors.